mkosi

```diff
@@ -2920,6 +2921,11 @@ def process_setting(args: CommandLineArguments, section: str, key: str, value: A
         elif key == "BuildSources":
             if args.build_sources is None:
                 args.build_sources = value
+        elif key == "CopySourcesMethod":
```

Comment:
maybe name this "SourceFileFiltering=" or so?

Reply:
or SourceFileAccess=? SourceFileTransfer=?

And then name the options `copy-all`, `copy-git-cached`, `copy-git-others`, `mount`?